Apromore-Custom-Plugins/Apromore-Editor/bpmn20xmlbasic/src/main/java/de/unihannover/se/infocup2008/bpmn/dao/ERDFDiagramDao.java

```diff
@@ -339,10 +337,8 @@ public void saveToWriter(Writer writer) {
         Result result = new StreamResult(writer);
 
         // Write the DOM document to the file
-        Transformer xformer;
         try {
-            xformer = TransformerFactory.newInstance().newTransformer();
-            xformer.transform(source, result);
+            transform(source, result);
         } catch (TransformerConfigurationException e) {
             // TODO Auto-generated catch block
             e.printStackTrace();
```

Comment:
Can we please log this exception instead of printing the stack trace.

Reply:
That's a good catch, and I would've fixed it if the comment had come before it was merged into development.  Fortunately this is dead code, since we don't use ERDF from this library, only BPMN.